### PR TITLE
USWDS-Site: Fix build errors

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Fixed build errors found in https://github.com/uswds/uswds-site/pull/2096
- Updated `package-lock.json` with newest updates from `develop` to repair errors found in [a11y_mobile](https://app.circleci.com/pipelines/github/uswds/uswds-site/4977/workflows/f4e9a2c3-538a-4a2a-b5bf-244f612a2ed4/jobs/18276) tests
- Updated broken/404 links to repair errors found  during [HTML Proofer checks](https://app.circleci.com/pipelines/github/uswds/uswds-site/4977/workflows/f4e9a2c3-538a-4a2a-b5bf-244f612a2ed4/jobs/18275).

Note: HTML Proofer will still flag errors related to `https://github.com/uswds/uswds/releases/tag/v3.5.0`. This will resolve once 3.5.0 is out.

## Related PRs
This should resolve some of the build errors found in https://github.com/uswds/uswds-site/pull/2096

## Preview link

Preview link: 
- [Social security number profile pattern](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-3.5.0-build-errors/patterns/create-a-user-profile/social-security-number/#references)
- [Progress easily](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-3.5.0-build-errors/patterns/complete-a-complex-form/progress-easily/#trauma-informed-design)

## Problem statement
Outdated links were causing [404 errors](https://app.circleci.com/pipelines/github/uswds/uswds-site/4973/workflows/532c92bb-167f-476c-adc7-f0154f241a97/jobs/18253).

## Problem statement
Removed any 404 links that were not related to 3.5.0. 

Please note that `npm run proof` will still flag `https://github.com/uswds/uswds/releases/tag/v3.5.0`. This will resolve once the release is...released.

## Testing and review
- Confirm the the updated links are suitable replacements
- Confirm that `npm run proof` only flags 3.5.0 release links

### CODE REVIEW

File: pages/patterns/create-a-profile/social-security-number.md [hunk 1]
```diff
@@ -116,12 +116,12 @@ Provide a single text field to enter the Social Security Number.
 ## References
 - 8 recommendations for creating effective input fields. (n.d.) Retrieved on July 20, 2022, from [https://baymard.com/learn/input-fields](https://baymard.com/learn/input-fields)
 - Consider using localized input masks for ‘phone’ and other restricted inputs. (November 28, 2017) Retrieved on July 20, 2022, from [https://baymard.com/blog/input-masking-form-field](https://baymard.com/blog/input-masking-form-field)
-- Masked field. (n.d.) Retrieved on July 20, 2022, from [https://design.cms.gov/components/masked-field](https://design.cms.gov/components/masked-field)
+- Masked field. (n.d.) Retrieved on July 20, 2022, from [https://design.cms.gov/v/5.0.2/components/masked-field](https://design.cms.gov/v/5.0.2/components/masked-field)
```
Comment (amyleadem) [inline pages/patterns/create-a-profile/social-security-number.md:119]: Could not find the masked field component in the current version, but found it in 5.0.2 for core. Proposing we point to that instead, but I am not sure if this is an accurate reference.

Comment (mahoneycm) [reply pages/patterns/create-a-profile/social-security-number.md:119]: I found a masked field in the latest version nested under their text field component!

https://design.cms.gov/components/text-field/masked-field/?theme=core

**Review by mahoneycm — COMMENTED** [review]
Tested the updated links and they're looking good! I found an updated link for masked field that points to the latest version of CMS design system if want to use that instead

File: pages/patterns/create-a-profile/social-security-number.md (hunk 1 above)
Comment (amyleadem) [reply pages/patterns/create-a-profile/social-security-number.md]: Perfect! Thanks, @mahoneycm

Updated in [93a4bf8](https://github.com/uswds/uswds-site/pull/2128/commits/93a4bf845d992e0d3344d9467b2e32b9231e8d53)

**Review by mahoneycm — APPROVED** [review]
Looks good! 

- [x] Updated links working and directing to expected pages
- [x] `npm run proof` only flags `3.5.0` links as expected

**Review by thisisdano — APPROVED** [review]

**Pull request merged**